plugins/com.aptana.core.io/src/com/aptana/core/io/vfs/Policy.java

```diff
@@ -43,5 +45,13 @@ public static void checkCanceled(IProgressMonitor monitor) {
 			throw new OperationCanceledException();
 	}
 
+	public static void error(int code, String message) throws CoreException {
+		error(code, message, null);
+	}
+
+	public static void error(int code, String message, Throwable exception) throws CoreException {
```

Comment:
Not sure why error handling should be part of the Policy - any other better class to keep these methods?

Reply:
Good point. Will see if we can push it to some other meaningful class.